angular/src/lib/accordion/accordion.component.ts

```diff
@@ -0,0 +1,267 @@
+
+import {
+  AfterContentInit, Component, ElementRef, Input, OnInit, Output, EventEmitter,
+  ContentChildren, QueryList, ChangeDetectorRef, Inject, forwardRef, ViewChild, AfterViewInit
+} from '@angular/core';
+
+import { AccordionService } from './accordion.service';
+
+@Component({
+  selector: 'cui-accordion-tab',
+  template: `
+    <div
+      [attr.aria-expanded]="isExpanded"
+      class="cui-accordion__group"
+      [ngClass]="tabWrapperClasses"
+    >
+      <div
+        #headerRef
+        role="button"
+        class="cui-accordion__header"
+        [ngClass]="headerClasses"
+        (click) = "toggle($event)"
+        (keydown)="onKeyDown($event)"
+        [attr.tabindex]="!disabled ? 0 : -1"
+      >
+        <div>{{header}}</div>
+        <span class="cui-accordion__header-icon"></span>
+      </div>
+
+      <div class="cui-accordion__content">
+        <div
+          class="cui-list-item"
+          id="cui-list-item-12"
+          role="listItem"
+          (click) = "handleContentClick($event)"
+        >
+          <ng-content></ng-content>
+        </div>
+      </div>
+    </div>
+  `,
+})
+export class AccordionTabComponent implements OnInit, AfterViewInit {
+
+  @Input() header: string = '';
+  /** @prop Set accordion tab to be expanded | false  */
+  @Input() isExpanded: boolean = false;
+  /** @prop Set the attribute disabled to the accordion tab | false */
+  @Input() disabled: boolean = false;
+  /** @prop Optional accordion tab css class string | '' */
+  @Input() tabClass: string = '';
+  /** @prop Set the height of the AccordionHeader to either the default or 56px | '' */
+  @Input() height: string = '';
```

Comment:
Will it be good to set the type of height to number here?

Reply:
agreed, changed to number